src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -1637,9 +1638,7 @@ public void testBlankRepositoryName() throws Exception {
     @Test
     public void testSubmoduleFixup() throws Exception {
         /* Unreliable on Windows and not a platform specific test */
-        if (isWindows()) {
-            return;
-        }
+        Assume.assumeFalse(isWindows());
```

Comment:
Running reliably on Windows in my tests without the Assume.  Let's check it on ci.jenkins.io
```suggestion
```